test/libsolidity/smtCheckerTests/functions/getters/array_of_structs_1.sol

```diff
@@ -0,0 +1,17 @@
+pragma abicoder v1;
+pragma experimental SMTChecker;
+
+struct Item {
+	uint x;
+	uint y;
+}
+
+contract D {
+	Item[][][] public items;
+
+	function test() public view returns (uint) {
+		// The autogenerated getters to not use ABI encoder.
```

Comment:
Not really sure what this sentence means

Reply:
It was in the original example posted but @bshastry. I also don't know what this means.

Reply:
Let's remove it then.

Reply:
Removed.